src/v2/pdf_resources/page_font.h

```diff
@@ -891,6 +879,37 @@ namespace pdflib
   }
 
 
+  void pdf_resource<PAGE_FONT>::init_font_matrix()
+  {
+    LOG_S(INFO) << __FUNCTION__;// << "\t" << json_font.dump(2);
+
+    std::vector<std::string> keys_0 = {"/FontMatrix"};
+
+    if(utils::json::has(keys_0, json_font))
+      {
+        //assert(subtype==TYPE_3);
+        auto result = utils::json::get(keys_0, json_font);
+
+        for(int d=0; d<6; d++)
+          {
+            font_matrix[d] = result[d].get<double>();
+          }
+      }
+    else
+      {
+        LOG_S(INFO) << "using default font-matrix";
+        font_matrix = {0.001, 0, 0, 0.001, 0, 0};
+      }
+
```

Comment:
Let's set an internal boolean flag here which analyses the font-matrix and avoids us to do `!=` downstream.